README.md

```diff
@@ -137,6 +137,24 @@ To use the Docker image, run:
 
 	docker run --rm -v $PWD:/mnt -w /mnt my:tag <shfmt arguments>
 
+### pre-commit
+
+It is possible to use shfmt with [pre-commit][pre-commit] and a `local`
+repo configuration like:
+
+```yaml
+  - repo: local
+    hooks:
+      - id: shfmt
+        name: shfmt
+        minimum_pre_commit_version: 2.4.0
+        language: golang
+        additional_dependencies: [mvdan.cc/sh/v3/cmd/shfmt@v3.1.1]
+        entry: shfmt
```

Comment:
I don't quite get this line either

Reply:
It's the executable run by the hook. https://pre-commit.com/#new-hooks

Reply:
Like `name`, it feels redundant, no? `mvdan.cc/sh/v3/cmd/shfmt@v3.1.1` is already enough to run the binary, like `go run mvdan.cc/sh/v3/cmd/shfmt@v3.1.1`.

I know this is not something you can fix right now. This is directed at upstream more than anything. IMO these fields should be inferred when not explicit.